libs/template/templates/default-python/template/{{.project_name}}/databricks.yml.tmpl

```diff
@@ -0,0 +1,58 @@
+# This is a Databricks asset bundle definition for {{.project_name}}.
+# See https://docs.databricks.com/dev-tools/bundles/index.html for documentation.
+bundle:
+  name: {{.project_name}}
+
+include:
+  - resources/*.yml
+
+artifacts:
+  {{.project_name}}:
+    type: whl
+    path: "."
+    build: "python3 setup.py bdist_wheel"
```

Comment:
This can be removed and the wheel will be auto-detected because of `setup.py`.

cc @andrewnester

Reply:
Is that what we want to recommend to customers, though? Then they have no path to import a wheel file from a scratch notebook or other unsupported location.

Reply:
Discussed offline. Block removed!